authapi/models.py

```diff
@@ -1 +1,19 @@
-# Create your models here.
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class SeedOrganization(models.Model):
+    # TODO: Implement SeedOrganization model
+    users = models.ManyToManyField(User)
+
+
+class SeedPermission(models.Model):
+    # TODO: Implement SeedPermission model
+    pass
+
+
+class SeedTeam(models.Model):
+    # TODO: Implement SeedTeam model
+    organization = models.ForeignKey(SeedOrganization)
+    users = models.ManyToManyField(User)
+    permissions = models.ManyToManyField(SeedPermission)
```

Comment:
Ah, I see, thanks for looking into it. I think the current approach sounds cleanest, +1.